apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -501,7 +501,7 @@ type ResourcePlacementStatus struct {
 
 	// +kubebuilder:validation:MaxItems=100
 
-	// FailedPlacements is a list of all the resources failed to be placed to the given cluster.
+	// FailedPlacements is a list of all the resources failed to be placed to the given cluster or the resource is unavailable.
 	// Note that we only include 100 failed resource placements even if there are more than 100.
 	// This field is only meaningful if the `ClusterName` is not empty.
 	// +optional
```

Comment:
when the resource is unavailable (later not ready), it will be populated as part of FailedPlacements. @ryanzhang-oss 
 @circy9 WDYT?